lib/build.gradle

```diff
@@ -20,7 +20,7 @@ plugins {
 }
 
 group 'com.dynatrace.metric.util'
-version = '1.3.0'
+version = '1.3.1'
```

Comment:
`1.3.0` is not released, right? Just wondering since some `public` parts have changed from `1.3.0` to `1.3.1` and moved to another package.

Reply:
Since it was not released we can just use `1.3.0` here.